CONTRIBUTING.md

```diff
@@ -2,20 +2,21 @@
 
 We recommend to create a virtual environment to contribute to the client.
 
-Run the following to create a virtual environment, activate it and install dependencies:
+To create a virtual environment, activate it, and install dependencies, run the following shell code:
+
 ```shell
 python3 -m venv .venv
 source .venv/bin/activate
 pip install -r requirements-devel.txt
 ```
 
-The next time you open your shell, you can activate your virtual environment using `source .venv/bin/activate`
+To activate your virtual environment, run `source .venv/bin/activate`.
 
-To run local instance of Weaviate, we recommend using Docker (see https://weaviate.io/developers/weaviate/installation/docker-compose).
+The newest client versions sometimes require upcoming Weaviate core features. We recommend using Docker (see https://weaviate.io/developers/weaviate/installation/docker-compose) to run a local instance of the latest Weaviate core for client development.
```

Comment:
Upcoming sounds unreleased

Reply:
Yes, the core and client have both been developing ahead of the released server core.

Reply:
My point is that I don't think client features get released without a corresponding server version. @dirkkul pls lmk if that's incorrect.

Reply:
this is contributing, so for potential developers. I think it makes sense to have it in

Reply:
Then we should maybe point them towards running a dev setup directly.